terraform/README.md

```diff
@@ -5,16 +5,30 @@ the Testflinger server into a dev, staging, or production environment.
 For development, it is recommended to use Juju with Microk8s.
 
 
-## Deploying an Environment
-
-For purposes of this example, we will be deploying a **dev**
-environment.
+## Setting up microk8s and juju for development
 
-This assumes that you have an existing Juju + k8s environment.
 Microk8s makes a great environment for development and testing this,
 and you can find more information about deploying it and using it with
 juju in this howto: https://juju.is/docs/olm/microk8s
 
+However, the required steps are:
+
+```
+sudo snap install microk8s --channel=1.27-strict/stable
+sudo usermod -a -G snap_microk8s $USER
+sudo chown -f -R $USER ~/.kube
+newgrp snap_microk8s
+sudo microk8s enable hostpath-storage dns ingress
+sudo snap alias microk8s.kubectl kubectl
+sudo snap install juju --channel=3.1/stable
+mkdir -p ~/.local/share
```

Comment:
I don't recall ever having to do a lot of these steps, especially the one for .local/share. What's that for? Not a problem, just curious.

Reply:
That's a workaround that can be needed if you're using juju from the snap confined

Reply:
The other steps are similarly probably only needed if running microk8s confined.

Reply:
ah, ok. Makes sense